superdesk/editor_utils.py

```diff
@@ -392,7 +454,7 @@ class Editor3Content(EditorContent):
     editor_version = 3
     HTML_EXPORTER = DraftJSHTMLExporter
 
-    def __init__(self, item, field='body_html'):
+    def __init__(self, item, field='body_html', html=True):
```

Comment:
we can guess content type from the suffix (body_**html**), so maybe we can detect with suffix by default with a `html=None` value, and use `True`/`False` only when we want to force or not HTML.

Reply:
we could guess it for core fields, but not sure about extra fields so would rather make it explicit